ApiDoctor.Publishing/CSDL/csdlwriter.cs

```diff
@@ -1727,35 +1727,25 @@ private static void AddRestrictionAnnotations(Dictionary<string, Annotations> an
                         stringBuilder.Append(value);
                     }
                 }
+
                 IODataAnnotatable annotatable = new Property();
                 var path = stringBuilder.ToString();
                 if (path.Equals(target, StringComparison.OrdinalIgnoreCase))
                 {
                     annotatable = (IODataAnnotatable)set;
                 }
                 var verb = method.HttpMethodVerb();
-                switch (verb)
-                {
-                    case "GET" when url.EndsWith("}", StringComparison.OrdinalIgnoreCase):
-                        AddReadByKeyRestriction(annotatable, method);
-                        break;
-                    case "GET":
-                        AddRestrictionAnnotation(annotatable, method, Term.ReadRestrictionTerm);
-                        break;
-                    case "POST":
-                        AddRestrictionAnnotation(annotatable, method, Term.InsertRestrictionsTerm);
-                        break;
-                    case "PUT":
-                    case "PATCH":
-                        AddRestrictionAnnotation(annotatable, method, Term.UpdateRestrictions);
-                        break;
-                    case "DELETE":
-                        AddRestrictionAnnotation(annotatable, method, Term.DeleteRestrictions);
-                        break;
-                    default:
```

Comment:
If we can, lets keep the handling of the default case. HTTP methods are technically more than 5 listed so it may be good to keep it incase the code comes ever across the others.